.github/workflows/continuous-monitoring.yml

```diff
@@ -29,14 +29,14 @@ jobs:
             aws cloudwatch put-metric-data --metric-name XRayJavaAgentGithubDistributionUnavailability --dimensions failure=rate --namespace MonitorAgent --value 0 --timestamp $(date +%s)
           fi
       
-      - name: Get Latest tag
-        id: latest_tag
-        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+      - name: Gets latest created release info
```

Comment:
I don't think fetching the release via a third party action is a good approach (especially for the workflow verifying distribution channel!). 

Can you do something like this (using Gradle's mechanisms for resolving latest version)?
https://github.com/aws/aws-xray-sdk-java/blob/master/smoke-tests/build.gradle.kts#L9

Reply:
Yep. I am going to close this one and create an issue to remove third party action for getting the tag from monitoring workflow.